shared-libs/rules-engine/test/target-state.spec.js

```diff
@@ -181,7 +181,8 @@ describe('target-state', () => {
       targetState.storeTargetEmissions(state, [], [
         mockEmission({ _id: 'a', groupBy: '1', pass: true }),
         mockEmission({ _id: 'b', groupBy: '2', pass: false }), // pass should have no effect
-        mockEmission({ _id: 'c', groupBy: '1' }),
+        mockEmission({ _id: 'c', groupBy: '1', pass: true }),
+        mockEmission({ _id: 'd', groupBy: '1', pass: false }),
```

Comment:
Please add another group here that has 2 emissions, one passing and one failing, to prove that this group doesn't count as passed.